File: src/main/java/dev/slne/discord/ticket/result/TicketCreateResult.java
package dev.slne.discord.ticket.result;

/**
 * The enum Ticket create result.
 */
public enum TicketCreateResult {

	/**
	 * The ticket was created successfully
	 */
	SUCCESS,

	/**
	 * The ticket already exists
	 */
	ALREADY_EXISTS,

	/**
	 * The guild was not found
	 */
	GUILD_NOT_FOUND,

	/**
	 * The guild config was not found
	 */
	GUILD_CONFIG_NOT_FOUND,
	
	/**
	 * The category was not found
	 */
	CATEGORY_NOT_FOUND,

	/**
	 * The channel could not be created
	 */
	ERROR,

	/**
	 * The user is missing permissions
	 */
	MISSING_PERMISSIONS

}

File: src/main/kotlin/dev/slne/discordold/ticket/result/TicketCreateResult.kt
package dev.slne.discordold.ticket.result

enum class TicketCreateResult {
    SUCCESS,
    ALREADY_EXISTS,
    GUILD_NOT_FOUND,
    GUILD_CONFIG_NOT_FOUND,
    CHANNEL_NOT_FOUND,
    ERROR,
    MISSING_PERMISSIONS,
    AUTHOR_NOT_FOUND,
    ROLE_NOT_FOUND,
}
